datasets/russian_super_glue/README.md

```diff
@@ -0,0 +1,673 @@
+---
+pretty_name: Russian SuperGLUE
+annotations_creators:
+- crowdsourced
+- expert-generated
+language_creators:
+- crowdsourced
+- expert-generated
+languages:
+- ru-RU
+licenses:
+- mit
+multilinguality:
+- monolingual
+size_categories:
+- 100K<n<1M
+- 1M<n<10M
+- 10M<n<100M
+- 100M<n<1B
+source_datasets:
+- original
+task_categories:
+- text-classification
+task_ids:
+- natural-language-inference
+- multi-class-classification
+---
+
+# Dataset Card for [Russian SuperGLUE]
+
+## Table of Contents
+- [Table of Contents](#table-of-contents)
+- [Dataset Description](#dataset-description)
+  - [Dataset Summary](#dataset-summary)
+  - [Supported Tasks and Leaderboards](#supported-tasks-and-leaderboards)
+  - [Languages](#languages)
+- [Dataset Structure](#dataset-structure)
+  - [Data Instances](#data-instances)
+  - [Data Fields](#data-fields)
+  - [Data Splits](#data-splits)
+- [Dataset Creation](#dataset-creation)
+  - [Curation Rationale](#curation-rationale)
+  - [Source Data](#source-data)
+  - [Annotations](#annotations)
+  - [Personal and Sensitive Information](#personal-and-sensitive-information)
+- [Considerations for Using the Data](#considerations-for-using-the-data)
+  - [Social Impact of Dataset](#social-impact-of-dataset)
+  - [Discussion of Biases](#discussion-of-biases)
+  - [Other Known Limitations](#other-known-limitations)
+- [Additional Information](#additional-information)
+  - [Dataset Curators](#dataset-curators)
+  - [Licensing Information](#licensing-information)
+  - [Citation Information](#citation-information)
+  - [Contributions](#contributions)
+
+## Dataset Description
+
+- **Homepage:** https://russiansuperglue.com/
```

Comment:
This website is unreachable for me

Reply:
Could you check again? Might've been something temporary. Currently, I can access it and I don't believe there are any regional restrictions. I'll need more details if it still is unreachable for you.

Reply:
It's working now, thanks :)